.ipynb_checkpoints/BANA Assignment iteration 1-checkpoint.ipynb

```diff
@@ -0,0 +1,2063 @@
+{
```

Comment:
Why is there a whole new file? Didn't we say to keep to one file?

If this file is like a different file the I can understand but this looks like the same as the first one